SavedInstances/SavedInstances.lua

```diff
@@ -2893,28 +2930,54 @@ function core:RefreshDailyWorldQuestInfo()
     local title = GetQuestLogTitle(GetQuestLogIndexByID(BountyInfo.questID))
     local timeleft = C_TaskQuest.GetQuestTimeLeftMinutes(BountyInfo.questID)
     local _, _, isFinish, questDone, questNeed = GetQuestObjectiveInfo(BountyInfo.questID, 1, false)
-    if not isFinish and timeleft then
+    if timeleft then
       if timeleft > 2880 then
-        t.DailyWorldQuest.days2 = {}
+        if t.DailyWorldQuest.days2 then else t.DailyWorldQuest.days2 = {} end
```

Comment:
```if x then else y``` is the same as ```if not x then y```...I just don't like using ```not``` (only explanation I can come up with for that).  When I was writing that part of the code I was getting errors without that loop, though I don't recall what the final reason was.